reactor-core/src/main/java/reactor/core/scheduler/Scheduler.java

```diff
@@ -101,16 +102,20 @@ default long now(TimeUnit unit) {
 	}
 	
 	/**
-	 * Creates a worker of this Scheduler that executed task in a strict
-	 * FIFO order, guaranteed non-concurrently with each other.
+	 * Creates a worker of this Scheduler.
 	 * <p>
 	 * Once the Worker is no longer in use, one should call dispose() on it to
 	 * release any resources the particular Scheduler may have used.
 	 * 
-	 * <p>Tasks scheduled with this worker run in FIFO order and strictly non-concurrently, but
-	 * there are no ordering guarantees between different Workers created from the same
-	 * Scheduler.
-	 * 
+	 * <p>Tasks scheduled with this worker are not guaranteed to run in FIFO order and
```

Comment:
I'd remove that sentence from the generic interface and put it in the javadoc of `fromExecutor`/`fromExecutorService`